automation/release_notes/generate.py

```diff
@@ -123,14 +125,21 @@ def _generate_release_notes_from_commits(
 
 #### Pull requests:
 {commits_for_pull_requests}
+
+#### Failed parsing:
+{failed_commits}
         """
         )
+        raise ValueError(failed_parsing_commits)
```

Comment:
Let's have a meaningful message here and not just the commits again
Something like `"Failed parsing some of the commits, added them at the end of the release notes"`